File: 01-02-2024/temperatura/ConversaoTemperatura.java
import java.util.Scanner;

public class ConversaoTemperatura {
    
    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {
            System.out.println("Informe a temperatura em ºC:");
            final int celsius = scanner.nextInt();

            System.out.format(
                "%sºC equivalem a %sºF", 
                celsius, 
                fahrenheit(celsius)
            ).println();
        }
    }

    private static double fahrenheit(int celsius) {
        return (celsius * 1.8) + 32;
    }

}

File: 01-02-2024/temperatura/conversao_temperatura.c
#include <stdio.h>

float fahrenheit(int celsius) {
    return (celsius * 1.8) + 32;
}

int main() {
    int celsius;

    printf("Informe a temperatura em ºC: ");
    scanf("%d", &celsius);

    printf("%dºC é equivalente a %.2fºF\n", celsius, fahrenheit(celsius));

    return 0;
}